repos/fdbt-site/cypress_tests/cypress/support/helpers.ts

```diff
@@ -344,10 +344,12 @@ export const randomlyDeterminePurchaseType = (isOtherProduct?: boolean): void =>
     continueButtonClick();
 };
 
-export const randomlyDecideTimeRestrictions = (): void => {
+export const randomlyDecideTimeRestrictions = (isEditing?: boolean): void => {
     let timeRestriction = 'N/A';
     if (getRandomNumber(0, 1) === 0) {
-        clickElementById('valid-days-not-required');
+        if (isEditing) {
+            clickElementById('valid-days-not-required');
```

Comment:
is this saying that if its editing a product, it will always select no? What about if the initial product had already selected no?